portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,91 @@
-#content {
+*{
+    font-family: Helvetica;
+}
+
+#header {
+  display: flex;
+  flex-direction: row;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  justify-content: space-between;
+  align-items: center;
+  width: 75%;
+}
+
+#header h1{
+  font-weight: 600;
+}
+
+#header ul{
+    display: flex;
+    flex-direction: row;
+    list-style:none;
+}
+
+#header a{
+    text-decoration: none;
+    color: #000;
+    padding-right: 35px;
+    font-size: 18px;
+    font-weight: 200;
```

Comment:
In the spirit of following the style guide we should alphabetize declarations.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Made fixes including, alphabetization, reducing indentations to 2 spaces, making hexadecimal color codes lowercase, adding semicolons to every line, and including multimedia fallbacks.